roles/openshift_aws/tasks/setup_master_instances.yml

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
This file seems to simply rename setup_master_group.yml, which is fine, but setup_master_group.yml appears to also still be present and unused.  Ensure setup_master_group.yml is cleaned up.